src/ert/ensemble_evaluator/_ensemble.py

```diff
@@ -29,12 +29,7 @@
 from ._wait_for_evaluator import wait_for_evaluator
 from .config import EvaluatorServerConfig
 from .identifiers import EVTYPE_ENSEMBLE_FAILED, EVTYPE_ENSEMBLE_STARTED
-from .snapshot import (
-    ForwardModel,
-    RealizationSnapshot,
-    Snapshot,
-    SnapshotDict,
-)
+from .snapshot import ForwardModel, RealizationSnapshot, Snapshot, SnapshotDict
```

Comment:
unrelated change? (will cause conflict with jonathans pr..)

Reply:
will turn it back